### ISSUE TITLE
Dropdown menus not working on the pop-up notification on Wayland

### ISSUE THREAD

**Issue by Gojkira** [evilsocket/opensnitch#733]
When there's a pop up notification that asks if the user wants to allow/deny a network request, I am unable to use the dropdown menus. The menus themselves are shown (eg. I can click on the arrow and view the options once, 30s, 1 min, 5 min etc.) but the buttons do not respond when clicked on. This only became an issue when I moved from Xorg/i3 to Wayland/Sway. I can still accept/deny requests and check/uncheck the checkboxes.

Edit: I just checked and this issue actually affects the entire UI, not just the pop-up notifications.

 - OpenSnitch version: 1.6.0-rc1 [Downloaded from AUR]
 - OS: Artix
 - Window Manager: Sway
 - Kernel version: 5.18.16
- Display Server: Wayland

**Comment by gustavo-iniguez-goya** [comment]
Hi @Gojkira , do other Qt/PyQt applications work? For example qbittorrent, or this examle of a combo box: 
[pyqt5-combobox.py.txt](https://github.com/evilsocket/opensnitch/files/9485320/pyqt5-combobox.py.txt)

```
$ mv pyqt5-combobox.py.txt pyqt5-combobox.py
$ python3 pyqt5-combobox.py
```

Selecting an item of the combo box should output text to the terminal.

As far as I know we're compatible with Wayland (cinnamon, xfce/opensuse, kde/opensuse, gnome/fedora, */arch). The dropdown menus are regular Qt widgets (QComboBox), so I'd expect errors with others apps based on Qt or PyQt.

The Arch wiki has some notes about this, and they mention Sway: https://wiki.archlinux.org/title/Wayland#Qt
So try installing `qt5-wayland`, and execute the GUI with the options mentioned:

`$ QT_QPA_PLATFORM="wayland;xcb" opensnitch-ui` or `$ QT_QPA_PLATFORMTHEME=qt5ct opensnitch-ui` (or a combination of both)

**Comment by Gojkira** [comment]
>do other Qt/PyQt applications work?

Kdenlive and qBittorrent don't, KeepassXC does although not perfectly. Sorry I think you're right this is a qt issue, if I had known I wouldn't have posted here. I guess I just never used combo boxes for other software so I didn't notice.

>pyqt5-combobox.py.txt

This behaves the exact same way. I can view all of the options but not click on them.

>qt5-wayland

I already had this installed. I also installed qt6-wayland but it didn't change anything.

>QT_QPA_PLATFORM="wayland;xcb"
>QT_QPA_PLATFORMTHEME=qt5ct 

I already had these in my bash_profile . When I use the entire commands you provided, nothing changed. When I just used 'xcb' the combo boxes worked fine. I might just use this until I find out why they're not working natively in wayland.
